test/performance/load-test/main.go

```diff
@@ -31,12 +31,18 @@ import (
 	netv1alpha1 "knative.dev/serving/pkg/apis/networking/v1alpha1"
 
 	"knative.dev/pkg/test/mako"
+	"knative.dev/pkg/test/mako/alerter"
 	"knative.dev/serving/pkg/apis/serving"
+	"knative.dev/serving/test/performance"
 	"knative.dev/serving/test/performance/metrics"
 )
 
 var (
-	flavor = flag.String("flavor", "", "The flavor of the benchmark to run.")
+	flavor          = flag.String("flavor", "", "The flavor of the benchmark to run.")
+	githubToken     = flag.String("github-token", "", "The path of github token")
+	slackUserName   = flag.String("slack-user-name", "", "The user name of the slack bot")
```

Comment:
any reason this needs to be a flag? I dont see this changing between tests and maybe we can move this to const?

Reply:
Done, moved to `performance.go`